DESCRIPTION

```diff
@@ -27,7 +27,7 @@ Imports:
     htmltools (>= 0.5.1.1)
 Suggests: 
     covr,
-    knitr,
+    knitr (>= 1.31),
```

Comment:
This is to use `knitr::pandoc_to()` - we could do `knitr:::pandoc_to()` or directly `knitr::opts_knit$get("rmarkdown.pandoc.to")` if we don't want that.